locales/en.default.json

```diff
@@ -121,6 +121,11 @@
         "delete": "Delete",
         "text-above": "",
         "text-below": ""
+      },
+      "confirm-delete-modal": {
+        "ok": "YES",
+        "cancel": "NO",
+        "message": "Please confirm that you want to delete the user \"{0}\"."
```

Comment:
I think we should do something like this for confirmation (at least change phrases)
![image](https://user-images.githubusercontent.com/22875828/76709799-5284f300-670a-11ea-828d-1dba179ca92b.png)

Reply:
Agree with Ilya

Reply:
Done
https://virtocommerce.atlassian.net/wiki/spaces/ODT/pages/216006668/Manage+Users